ipam/claim.go

```diff
@@ -78,28 +78,36 @@ func (c *claim) Try(alloc *Allocator) bool {
 		return false
 	}
 
-	if c.ident == api.NoContainerID {
-		c.ident = c.cidr.Addr.String()
-	}
 	// We are the owner, check we haven't given it to another container
-	switch existingIdent := alloc.findOwner(c.cidr.Addr); existingIdent {
-	case "":
+	existingIdent := alloc.findOwner(c.cidr.Addr)
+	switch {
+	case existingIdent == "":
+		// Unused address, we try to claim it:
 		if err := alloc.space.Claim(c.cidr.Addr); err == nil {
-			alloc.debugln("Claimed", c.cidr, "for", c.ident)
-			alloc.addOwned(c.ident, c.cidr, c.isContainer)
+			ident := c.ident
+			if ident == api.NoContainerID {
+				ident = c.cidr.Addr.String()
+			}
+			alloc.debugln("Claimed", c.cidr, "for", ident)
+			alloc.addOwned(ident, c.cidr, c.isContainer)
 			c.sendResult(nil)
 		} else {
 			c.sendResult(err)
 		}
-	case c.ident:
-		// same identifier is claiming same address; that's OK
-		alloc.debugln("Re-Claimed", c.cidr, "for", c.ident)
+	case (existingIdent == c.ident) || (c.ident == api.NoContainerID && existingIdent == c.cidr.Addr.String()):
+		// Same identifier is claiming same address, which is OK:
+		alloc.debugln("Re-claimed", c.cidr, "for", c.ident)
```

Comment:
Generally we keep trivial fixes to formatting, spelling, etc., outside of behaviour-changing PRs, so it's easier to review the change that matters.
I would challenge whether these two are necessary at all.